pkg/cache/tas_flavor_snapshot.go

```diff
@@ -414,3 +414,29 @@ func (s *TASFlavorSnapshot) notFitMessage(fitCount, totalCount int32) string {
 	}
 	return fmt.Sprintf("topology %q allows to fit only %v out of %v pod(s)", s.topologyName, fitCount, totalCount)
 }
+
+// lexSortDomains return order of two domains ordered by their levelValues
+func lexSortDomains(a, b *domain) int {
```

Comment:
let's move this function to a dedicated package (rename and accept string slices as suggested in another comment).

Reply:
Done